packages/editor/src/components/inserter/menu.js

```diff
@@ -269,7 +268,18 @@ export class InserterMenu extends Component {
 						</PanelBody>
 					}
 
-					<InserterResultsPortal.Slot fillProps={ { filterValue } } />
+					<Slot name="Inserter.InlineElements" fillProps={ { filterValue } }>
```

Comment:
I'm wondering if we should move this slot to its own file similar to what we had before. This would help to keep this file smaller.

Reply:
Moved to its own file with https://github.com/WordPress/gutenberg/pull/10209/commits/3bea61e1881199603f7f68b417389715c6213f07